lib/LaTeXML/resources/CSS/LaTeXML.css

```diff
@@ -59,7 +59,9 @@
 
 /* .ltx_chapter_title, etc should be in ltx-article.css etc.
  */
-.ltx_page_main { margin:0px; padding:1em 3em 1em 2em; }
+.ltx_page_main { margin:0px auto; max-width: 36em; max-width: 72ch;
```

Comment:
The effects of this snuck up on me: Is there a reason to specify max-width in by default (especially so small)?  To my tastes, it's unpleasant on the desktop with a large browser window, and doesn't seem to have any constructive effect in my limited testing on mobile.

Reply:
Sorry, I must have brought this over from my own stylesheet! Not sure how it ended up there. My bad. I agree that it shouldn't be the default, it all depends on your use case.

Reply:
Ok, I just checked what I was doing in January. I think I pushed this commit by semi-accident. I was preparing the margin change but I hadn't really finished. At least I made separate commits, so you can just revert aebd080119f4baaf9a89307069fdae556c0949b2 to limit the change to the min(...) tweak.

I'll review the padding change as well, because now I do not feel sure that was fully baked either. I think the sidebar is still broken on mobile. I should at least double check that it's not any worse than before.